e2e/tests/functional/plugins/telemetryTable/telemetryTable.e2e.spec.js

```diff
@@ -92,7 +92,7 @@ test.describe('Telemetry Table', () => {
     await page.goto(table.url);
 
     await page.getByRole('searchbox', { name: 'message filter input' }).click();
-    await page.getByRole('searchbox', { name: 'message filter input' }).fill('Roger');
+    await page.keyboard.type('Roger', { delay: 150 });
```

Comment:
I think you could add coverage by using the previous logic and instantly changing the text. As long as the actions are quick enough, you should be able to cover the filterChanged codepath, no?
```js
    await page.getByRole('searchbox', { name: 'message filter input' }).fill('Roger2');
    await page.getByRole('searchbox', { name: 'message filter input' }).fill('Roger');
```

Reply:
I guess the question to answer would be, does this test fail if we revert the functional changes? Otherwise we should just keep the `fill()` since it won't matter and we'll save a sliver of runtime in the test.

Reply:
@scottbell did you catch my meaning?

Reply:
@unlikelyzero I tested using both ways, and both are firing `filterChanged` FWIW, as the table is filtering values. I'm not sure why our code coverage tool isn't reporting it!